.github/workflows/twister.yaml

```diff
@@ -13,7 +13,7 @@ on:
       - collab-*
   schedule:
     # Run at 03:00 UTC on every Sunday
```

Comment:
Please update :)

Reply:
and thats why comments are useless....

Reply:
drop it? we know how to read crontabs, and we know how to get to the manpage that explains how to read crontabs

Reply:
Not everyone picked cron as their second language in highschool though :)

Reply:
[cronguru](https://crontab.guru/) for the rescue :)

Reply:
replace with "man 5 crontab"